guides/plugins/plugins/plugin-base-guide.md

```diff
@@ -70,7 +70,7 @@ Basically that's it for the PHP part, your basic plugin class is already done.
 {% hint style="info" %}
 Here's a video showing how to bootstrap a plugin from our free online training ["Backend Development"](https://academy.shopware.com/courses/shopware-6-backend-development-with-jisse-reitsma).
 
-**[Creating a plugin](https://www.youtube.com/watch?v=_Tkoq5W7woI)**
+Also, refer to this video on **[Creating a plugin](https://www.youtube.com/watch?v=_Tkoq5W7woI)**
```

Comment:
Not sure what this does - the text above refers to the link below.

```suggestion
**[Creating a plugin](https://www.youtube.com/watch?v=_Tkoq5W7woI)**
```

Reply:
Rephrased it as " Refer to this video on **[Creating a plugin](https://www.youtube.com/watch?v=_Tkoq5W7woI)** that shows how to bootstrap a plugin. Also available on our free online training ["Shopware 6 Backend Development"](https://academy.shopware.com/courses/shopware-6-backend-development-with-jisse-reitsma)." 

This includes necessary redirect to the video and also a mention about our free online training.